Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -1706,6 +1716,8 @@ class PackageCommandTestCase: CommandsBuildProviderTestCase {
     }
 
     func testOnlyUseVersionsFromResolvedFileFetchesWithExistingState() async throws {
+       try XCTSkipOnWindows(because: "error: Package.resolved file is corrupted or malformed, needs investigation") 
```

Comment:
issue: can we raise a GitHub issue to fix this test, and reference the issue in the reason?  

This comment applies to all skipped tests and can be done in a follow-up PR.